param/obstacle_detector.yaml

```diff
@@ -0,0 +1,15 @@
+min_x: -5.0
```

Comment:
Have one comment saying what units these are in and another saying what cluster_threshold is. Everything else in this file is pretty self-evident.

Reply:
Also, because this shows up under Andrew's comment - get_rotation_matrices is especially indecipherable

Reply:
Fixed

Reply:
Haha, I had to google mks units and I'm surprised I hadn't heard of that before. Thank you!